desc/objectives/_coils.py

```diff
@@ -1285,10 +1492,10 @@ class ToroidalFlux(_Objective):
 
     Parameters
     ----------
-    eq : Equilibrium
-        Equilibrium for which the toroidal flux will be calculated.
-        The Equilibrium is assumed to be held fixed when using this
-        objective.
+    eq : Equilibrium or FourierRZToroidalSurface
```

Comment:
General comment, why don't we change the argname to something like `thing`? For objectives that can take multiple different class in addition to `Equilibrium`.

Reply:
we hesitate to change existing objective argnames in order to not break existing code using it